packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -419,20 +419,32 @@ export class AtomicSearchBox {
     this.ariaMessage = '';
   }
 
+  private makeSuggestionPart(isSelected: boolean) {
+    let part = 'suggestion';
+    if (isSelected) {
+      part += ' active-suggestion';
+    }
+    return part;
+  }
+
   private renderSuggestion(
     suggestion: SearchBoxSuggestionElement,
-    index: number
+    index: number,
+    lastIndex: number
   ) {
     const id = `${this.id}-suggestion-${index}`;
     const isSelected = id === this.activeDescendant;
+    if (suggestion.key === 'recent-query-clear' && index === lastIndex) {
```

Comment:
As was discussed in scrum I don't think we should do this.

There should be a mechanism for suggestion provider to mark an element as an "header". 

And then the search box will be able to decide wether to render the header or not when it hit that corner case.